src/mwi/get_ecmwf_seasonal_data.py

```diff
@@ -47,7 +47,7 @@ def main(download=False, compute_stats=True, use_cache=True):
         use_unrounded_area_coords=USE_UNROUNDED_AREA_COORDS
     )
     df_country_boundaries = gpd.read_file(ADM0_BOUND_PATH)
-    if download:
+    if download and SOURCE_CDS:
```

Comment:
This might be confusing because if the user passes `download=True` they will not know why the download isn't working. Maybe provide an explanation message? (Or at least a comment)